src/inference/src/dev/core_impl.cpp

```diff
@@ -920,7 +921,9 @@ ov::SoPtr<ov::ICompiledModel> ov::CoreImpl::load_model_from_cache(
 
             compiled_model = context._impl ? plugin.import_model(networkStream, context, config)
                                            : plugin.import_model(networkStream, config);
-            compiled_model->loaded_from_cache();
+            if (auto wrapper = std::dynamic_pointer_cast<InferenceEngine::ICompiledModelWrapper>(compiled_model._ptr)) {
+                wrapper->get_executable_network()->loadedFromCache();
```

Comment:
I suppose we also need to update `import_model` methods to set this property?

Reply:
Updated